src/control/cmd/dmg/telemetry.go

```diff
@@ -379,6 +385,24 @@ func (cmd *metricsListCmd) Execute(args []string) error {
 	return nil
 }
 
+func getMetricsHost(cfg *control.Config, custom []string) (string, error) {
+	var hostlist []string
+	if len(custom) > 0 {
+		hostlist = custom
+	} else {
+		hostlist = cfg.HostList
```

Comment:
Hmm. I'm concerned that this is going to be another source of confusion, as most "real" configs will have > 1 host in the list and so this will most often result in an error, if I'm understanding the code correctly.

I think what might be least surprising is that if the admin does not supply an explicit hostlist, we just default to localhost. That happens to be the default if an empty config is used, but it might be better to make it explicit so that it works the same for everyone regardless of what their config is.

Reply:
Makes sense to me. I'll fix it.